opwen_email_server/jobs/store_inbound_emails.py

```diff
@@ -1,6 +1,7 @@
 from opwen_email_server.api import email_receive
 from opwen_email_server.backend import server_datastore
 from opwen_email_server.services.queue_consumer import QueueConsumer
+from opwen_email_server.utils.email_parser import convert_img_url_to_base64
```

Comment:
Should this message be called `inline_images` instead? In that way, the implementation details ("base64 conversion") would be encapsulated in case they might change in the future.